src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -1381,7 +1381,7 @@ def _pipeline(r: Redis) -> RedisPipeline:
             desired_session_count = endpoint.desired_session_count
             if (
                 endpoint.lifecycle_stage == EndpointLifecycle.DESTROYING
-                and len(endpoint.routings) == 0
+                and len(non_error_routings) == 0
```

Comment:
Minor fix. I'd recommend renaming this variable to "active" voice, like "active_routings".

Reply:
`successfully_created_routings` looks good.